webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/RegistrationCommandsPageSelectors.js

```diff
@@ -9,11 +9,24 @@ import {
   selectAPIResponse,
 } from '../../../redux/API/APISelectors';
 
+import { STATUS } from '../../../constants';
+
 // Form API Data
 
 export const selectAPIStatusData = state =>
   selectAPIStatus(state, REGISTRATION_COMMANDS_DATA);
 
+export const selectApiDataErrorCode = state => {
+  const isError =
+    selectAPIStatus(state, REGISTRATION_COMMANDS_DATA) === STATUS.ERROR;
+
+  return (
+    (isError &&
+      selectAPIResponse(state, REGISTRATION_COMMANDS_DATA).response.status) ||
+    null
+  );
+};
+
 export const selectOrganizations = state =>
   selectAPIResponse(state, REGISTRATION_COMMANDS_DATA).organizations || [];
```

Comment:
Not for this PR, but I wonder why this doesn't use `selectAPIStatusData(state)`
```suggestion
  selectAPIStatusData(state).organizations || [];
```

Reply:
`selectAPIStatusData` is returning the state only using `selectAPIStatus`

```js
export const selectAPIStatus = (state, key) =>
  selectAPIByKey(state, key).status;
```